dask/dataframe/io/tests/test_parquet.py

```diff
@@ -160,6 +170,18 @@ def write_read_engines(**kwargs):
     fp_pandas_xfail = write_read_engines()
 
 
+@PYARROW_MARK
+def test_pyarrow_getengine():
+    # Check that the default engine for "pyarrow"/"arrow"
+    # is not the `pyarrow.dataset`-based engine
```

Comment:
I don't think we want the "not", but it would be good for you to confirm 

```suggestion
    # is the `pyarrow.dataset`-based engine
```

Reply:
oops!